client/src/pages/ProfileSetting/ProfileSetting.tsx

```diff
@@ -47,9 +43,43 @@ export default function ProfileSetting() {
   };
 
   return (
-    <Grid container className={classes.root}>
-      <AuthHeader linkTo="/messages" asideText="testing upload file page" btnText="open uploadFile page" />
-      <Box className={classes.tabContainer}>
+    <Box className={classes.root}>
+      <Tabs
+        orientation="vertical"
+        variant="standard"
+        value={value}
+        onChange={handleChange}
+        aria-label="Profile Settings"
+        className={classes.tabs}
+        indicatorColor="primary"
+      >
+        <Tab label="Profile" {...a11yProps(0)} />
+        <Tab label="Personal Information" {...a11yProps(1)} />
+        <Tab label="Payment Details" {...a11yProps(2)} />
+        <Tab label="Notifications" {...a11yProps(3)} />
+        <Tab label="Password" {...a11yProps(4)} />
+      </Tabs>
+      <TabPanel value={value} index={0}>
+        {value == 0 && <Profile />}
+      </TabPanel>
+      <TabPanel value={value} index={1}>
+        {value == 1 && <PersonalInformationForm />}
+      </TabPanel>
+      <TabPanel value={value} index={2}>
+        Payment Details
+      </TabPanel>
+      <TabPanel value={value} index={3}>
+        Notifications
+      </TabPanel>
+      <TabPanel value={value} index={4}>
+        Password Change
+      </TabPanel>
+    </Box>
+  );
+};
+
+{
+  /* <Box className={classes.tabContainer}>
```

Comment:
If we are not using this, can we please remove this commented code?

Reply:
Gone